models/Book.js

```diff
@@ -0,0 +1,12 @@
+const { Schema, model } = require('mongoose');
+
+const bookSchema = new Schema({
+  title: {type: String},
+  description: {type: String},
+  status: {type: String},
+  email:  {type: String},
+});
+
+const Book = model('Book', bookSchema);
```

Comment:
Nice work team!

Feedback here regarding submission (card 2/task list): Modularize this today by putting schema and model in separate files. 👍

Reply:
Please disregard previous comment :) I was taught to put the schema in one file, and model into another file (so they are separate from one another), so, after further review,  I am thinking that I read the card wrong and this way is just fine/meets requirements 👍